engine/src/main/java/org/terasology/rendering/cameras/PerspectiveCamera.java

```diff
@@ -49,8 +51,9 @@
 
     private Vector3f tempRightVector = new Vector3f();
 
-    public PerspectiveCamera(PerspectiveCameraSettings cameraSettings) {
-        this.cameraSettings = cameraSettings;
+    public PerspectiveCamera(WorldProvider worldProvider, RenderingConfig renderingConfig) {
+    	super(worldProvider, renderingConfig);
+        this.cameraSettings = renderingConfig.getCameraSettings();
```

Comment:
It might be due to tab characters. It could look right on your system but not here via diff or in other workspaces. Tweak at your IntelliJ code style settings for the project and make sure to use 4 spaces instead of tabs :-)